beacon-chain/db/kv/operations.go

```diff
@@ -18,6 +18,9 @@ func (kv *Store) VoluntaryExit(ctx context.Context, exitRoot [32]byte) (*ethpb.V
 		return nil, err
 	}
 	exit := &ethpb.VoluntaryExit{}
+	if len(enc) == 0 {
```

Comment:
nitpick, but could you do the length check above this, right after you retrieve `enc` from the method ?

Reply:
Sure thing, done!